src/test/java/com/parasoft/demoapp/graphql/GraphQLTestErrorType.java

```diff
@@ -4,7 +4,11 @@
 
 enum GraphQLTestErrorType {
     UNAUTHORIZED("Current user is not authorized."),
-    FORBIDDEN("Current user does not have permission.");
+    FORBIDDEN("Current user does not have permission."),
```

Comment:
We only put shared error here. So let's move these new errors back into the tests.